pkg/otlp/model/translator/metrics_translator_test.go

```diff
@@ -354,6 +354,33 @@ func TestMapIntMonotonicWithReboot(t *testing.T) {
 	)
 }
 
+func TestMapIntMonotonicReportFirstValue(t *testing.T) {
```

Comment:
Should we add a test case where the cache already has some data about the timeseries, to test that we send the diff in this situation?